packages/api/src/models/animalUnionBatchInternalIdentifierModel.js

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import Model from './baseFormatModel.js';
```

Comment:
As mentioned above, it would be nice if there was some way (and I think changing the view name is probably best, but at least a code comment here) to make it clear that this is not a table.